lib/matplotlib/dviread.py

```diff
@@ -1030,6 +1030,12 @@ def find_tex_file(filename, format=None):
         The library that :program:`kpsewhich` is part of.
     """
 
+    if six.PY3:
+        if isinstance(filename, bytes):
+            filename = filename.decode('utf-8', errors='replace')
```

Comment:
Should probably use https://docs.python.org/3/library/os.html#os.fsdecode here?
For format I dunno what is the correct encoding; that probably is deep into the kpsewhich sources but I guess all relevant formats have ASCII compatible names so utf-8 should just be fine.

Reply:
then perhaps add a comment that in practice the encoding is irrelevant -- other than being ASCII-compatible -- because you don't expect non-ASCII filenames to be found.